docs/modules/ROOT/pages/servlet/authentication/logout.adoc

```diff
@@ -24,6 +24,8 @@ When you include {spring-boot-reference-url}using.html#using.build-systems.start
 If you request `GET /logout`, then Spring Security displays a logout confirmation page.
 Aside from providing a valuable double-checking mechanism for the user, it also provides a simple way to provide xref:servlet/exploits/csrf.adoc[the needed CSRF token] to `POST /logout`.
 
+Please note that if xref:servlet/exploits/csrf.adoc[the CSRF protection] is disabled in configuration, no logout confirmation page is shown to the user and the logout is performed direclty.
```

Comment:
Will you please change `[the CSRF protection]` to `[CSRF protection]`?

Please also change `direclty` to `directly`.